app/src/main/java/com/github/codetanzania/ui/fragment/IssueDetailsFormFragment.java

```diff
@@ -133,6 +154,28 @@ private void handleDynamicUserEvents(final RadioGroup view) {
     private void bindData() {
         String textData = getArguments().getString(KEY_SELECTED_SERVICE);
         mEtIssueTitle.setText(textData);
+
+        // get cached issues
+        List<Open311Service> serviceList = Open311ServicesUtil.cached(getActivity());
+        mIssueCategoryDialogPicker = new IssueCategoryPickerDialog((ArrayList<Open311Service>) serviceList, mOnSelectIssueCategory);
```

Comment:
Does this need to be a member variable? Perhaps it is better to instead dynamically generate it when needed.